docs/src/modules/components/PickersPlayground.tsx

```diff
@@ -16,6 +16,7 @@ import MenuItem from '@mui/material/MenuItem';
 import Divider from '@mui/material/Divider';
 import Tooltip from '@mui/material/Tooltip';
 import Typography from '@mui/material/Typography';
+// eslint-disable-next-line no-restricted-imports
```

Comment:
We don't intend this to be used in production and have it in most of the Pickers demos (docs/data/date-pickers/**)
If it's possible, I would vote to add an exception for this import. 🤔

Reply:
👀 For some reason eslint only reports this instance